docs/styles.rst

```diff
@@ -0,0 +1,15 @@
+Styles
+======
+
+VSG supports several predefined styles.
+They can be accessed by using the **--style** command line option.
+
++---------------+--------------------------------------+
+| Configuration | Description                          |
++---------------+--------------------------------------+
+| legacy        | Rules based on the pre 2.0.0 release |
+| minimal       | Only indenting is checked            |
++---------------+--------------------------------------+
+
+These default configurations provide a base which can be further modified using the **--configuration** option.
```

Comment:
This suggests, that default configurations are not complete/finished, which will not be true. Their main goal is to use them, modifying/extending them is just an extra feature.

Reply:
I will elaborate this sentence.